apps/app-article/src/App.js

```diff
@@ -375,6 +391,10 @@ class App extends Component {
         this.fetchArticleFromApi(urlParams.get('uuid'));
     }
 
+    onUserFetchFail(error){
```

Comment:
What will happen here when you don't set/remove any cookies? `onUserFetchFail` is most likely invoked because the `token` is expired (this means the user is in fact logged out). How will this effect the native part?